### ISSUE TITLE
No easy way to know theme compatibility, some offered themes are obsolete

### ISSUE THREAD

**Issue by counterpoint** [getgrav/grav#3486]
Someone installing Grav afresh may want to install extra themes. The "Add" function shows a list of themes in alphabetical order. There is no indication of which themes are compatible with the current version of Grav. In fact, the first two themes shown are incompatible with current Grav, and are almost certainly obsolete and unsupported. A lot of time can be wasted attempting to find out which themes are even worth considering. It should be easier to select only compatible themes, and themes without current support should be removed.

**Comment by rhukster** [comment]
If you follow the guide to enable twig compatibility: https://learn.getgrav.org/17/advanced/grav-development/grav-17-upgrade-guide#twig

then all themes should work fine.

**Comment by simplycomputing** [comment]
I agree with @counterpoint. It's confusing for new users to have broken themes presented with no indication of compatibility. I have only been using Grav a short time but already there have been 2 posts on the forum from people asking why themes don't work. I realise they can be fixed by editing yaml files but this is in the "advanced" section of the docs and we are talking about first time users. The natural tendency is to load up Grav then immediately check out a few different themes.

The simplest solution would be to add a line at the top of the "available themes" page saying that not all themes may be compatible with this version and giving the link to fix.

**Comment by counterpoint** [comment]
Thanks @rhukster but if you are referring to the auto escape fix, it isn’t sufficient to make Aerial (the first theme offered) work. In fact the demo site no longer exists, and the theme is unchanged for 6 years. Surely, if Grav is to be usable by most people, it should not offer themes that do not work without needing to edit files using knowledge that isn’t readily obtained.

**Comment by pamtbaau** [comment]
I have created  discussion topic [Theme repository cleanup and quality control](https://github.com/getgrav/grav/discussions/3490) and gave some suggestions to go forward. Your feedback is welcome.
